src/js/angular/graphexplore/controllers/rdf-class-hierarchy.controller.js

```diff
@@ -448,10 +452,12 @@ function RdfClassHierarchyCtlr($scope, $rootScope, $location, $repositories, $li
     }
 
     function getClassHierarchyData() {
-
+        if (!$licenseService.isLicenseValid()) {
```

Comment:
I wonder why this was moved from the its previous place below? What is the difference? As far as I can see, the refreshDiagramExternalElements method doesn't do any requests.

Reply:
I agree. I'll move it back.